lib/util/swagger.js

```diff
@@ -0,0 +1,238 @@
+'use strict'
```

Comment:
Same question as the openapi utility file.

Reply:
Done